bugbot/rules/duplicate_copy_metadata.py

```diff
@@ -25,12 +27,24 @@
 
 
 class DuplicateCopyMetadata(BzCleaner):
+    def __init__(self, last_modification_days: int = 7):
+        """Constructor
+        Args:
+            last_modification_days: Number of days to look back for bugs that
+                were modified.
+        """
+        super().__init__()
+
+        self.last_modification_date = lmdutils.get_date("today", last_modification_days)
+
     def description(self):
         return "Copied fields from duplicate bugs"
 
     def handle_bug(self, bug, data):
-        bugid = str(bug["id"])
-        data[bugid] = bug
+        if bug["last_change_time_non_bot"] < self.last_modification_date:
```

Comment:
This change isn't directly related, right? Could you put it in a separate commit?

Reply:
It is related to the migration process. The migration script (i.e., https://github.com/mozilla-bteam/bmo/pull/2067) will modify all accessibility bugs. This will ensure we will not pick up old bugs updated by the migration script.

Reply:
I will split it into a separate commit to have a clearer history.